### ISSUE TITLE
IgxPivotGrid column headers are not keyboard accessible

### ISSUE THREAD

**Issue by ddaribo** [IgniteUI/igniteui-angular#15988]
## Description  
According to the Pivot Grid topic [Keyboard navigation section](https://www.infragistics.com/products/ignite-ui-angular/angular/components/pivotGrid/pivot-grid-features#keyboard-navigation), the navigation of the pivot grid is similar to the other grids.

However, the column headers (both the horizontal `igx-grid-header`-s and the vertical `igx-pivot-row-dimension-header`-s) cannot be reached via keyboard only. The header row area is a tab stop and within it the tabbable elements in the chips are also tab stops. However, the headers are not part of the tab navigation and therefore a keyboard-only user cannot reach them.

In the other grids, the header row is a tab stop, where a single header is activated, and on the next tab press - the grid body. 
 * igniteui-angular version: any
 * browser: any 

## Steps to reproduce  

1. Navigate the grid with the keyboard, i.e. [any demo](https://www.infragistics.com/angular-demos/pivot-grid/pivot-grid-layout) from the docs.

## Result  
The following gif demonstrates navigating the grid with Tab only:
![Image](https://github.com/user-attachments/assets/9189ddfc-0522-4db0-86ae-e83ed6c12153)

Actually, the behavior is a bit inconsistent, hence logging this as a bug. In case a header was activated by clicking, it would appear activated on tab navigation **_once_**, after which - not. However, it seems not to be the active node of the navigation, since the arrow keys to not work to navigate this headers at the point. In the below gif,  the `2007` header was clicked and then tab navigation started, and then, when returning with `Shift + Tab`, the header does not even appear activated :

![Image](https://github.com/user-attachments/assets/23962acc-63ea-4518-9910-dedbf519dd97)

## Expected result  
The pivot grid headers to be accessible by keyboard only. Could be as a tab stop, where a single header is activated, similar to other grids?

## Attachments  
Attach a sample if available, and screenshots, if applicable.
